addons/medical_treatment/XEH_preStart.sqf

```diff
@@ -1,3 +1,11 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+private _replacementItems = [];
+
+{
+    _replacementItems pushBack [configName _x, getArray _x];
+} forEach configProperties [configFile >> QEGVAR(medical,replacementItems), "isArray _x"];
+
+uiNamespace setVariable [QGVAR(replacementItems), _replacementItems];
```

Comment:
need to use `compileFinal`
or I think it would be fine to just move to preInit, this is going to take ~0.1ms to run